app/src/store.js

```diff
@@ -1,3 +1,7 @@
-import { createStore } from 'redux'
+import { createStore,applyMiddleware  } from 'redux'
```

Comment:
```suggestion
import { createStore, applyMiddleware  } from 'redux'
```

Is eslint activated?

Reply:
no, can I just merge the branch that Jessica created for the eslint-setup? I mean that's the point right?

Reply:
Yes exactly. You can do that

Reply:
> no, can I just merge the branch that Jessica created for the eslint-setup? I mean that's the point right?

Or you can pull develop, I have already merged the ESLint branch